package-lock.json

```diff
@@ -45,6 +45,7 @@
         "pretty-quick": "4.0.0",
         "rimraf": "5.0.5",
         "sinon": "19.0.2",
+        "sonarqube-scanner": "^4.2.4",
```

Comment:
I don't think that's needed and it's removed when I run `npm i`

Reply:
Good catch, my mistake